test/e2e/framework/config.go

```diff
@@ -49,6 +50,7 @@ func init() {
 }
 
 func registerFlags(c *testConfig) {
-	flag.StringVar(&c.kubeconfig, "kubeconfig", "", "Path to kubeconfig for a kcp server.")
-	flag.BoolVar(&c.useDefaultServer, "use-default-server", false, "Whether to use server configuration from .kcp/admin.kubeconfig.")
+	flag.StringVar(&c.kcpKubeconfig, "kcp-kubeconfig", "", "Path to the kubeconfig for a kcp server.")
+	flag.StringVar(&c.pclusterKubeconfig, "pcluster-kubeconfig", "", "Path to the kubeconfig for a kubernetes cluster to sync to.")
+	flag.BoolVar(&c.useDefaultKCPServer, "use-default-kcp-server", false, "Whether to use server configuration from .kcp/admin.kubeconfig.")
```

Comment:
Will need to update the e2e-shared-server job in `.github/workflows/ci.yaml` to reflect the change in the arg name.